landing-page/styles.css

```diff
@@ -0,0 +1,122 @@
+*{
+  margin:0;
+  padding:0;
+}
+body{
+  margin:0;
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-size: 17px;
+  line-height:1.6;
+}
+#simbol {
+  text-align: center;
+  color: black;
+  font: italic small-caps bold 30px Georgia, Garamond, serif;
+  background: limegreen;
+  margin: 10px;
+  border: 7px solid purple;
+  padding: 10px;
+
+}
+#showcase {
+  background-image:url('https://www.designcuts.com/wp-content/uploads/2018/11/30-Spring-Watercolor-Backgrounds-strip5.jpg');
+  background-size: cover;
+  background-position: center;
+  height:100vh;
+  display:flex;
+  flex-direction:column;
+  justify-content:center;
+  align-items:center;
+  text-align:center;
+  padding:0 20px;
+}
+#showcase h1 {
+  font-size:30px;
+  line-height:1.2;
+  }
+#showcase p{
+  font-size:20px;
+}
+#showcase .button1 {
+  font-size:18px;
+  text-decoration:none;
+  color: black;
+  border: black 1px solid;
+  padding:10px 20px;
+  border-radius:10px;
+  margin-top:15px;
+}
+#showcase .button1:hover {
+  background: darkcyan;
+  color:#fff;
+}
```

Comment:
Same idea about consistency: the html color names will work just fine alongside hex values, but it is more conventional to choose one way of referring to colors or another (and most people use hex or rgb, or rgba, values).